app/lib/clients/create_employees.rb

```diff
@@ -0,0 +1,54 @@
+class Clients::CreateEmployees
+  def initialize(employees, time_off)
+    @employees = employees
+    @time_off = time_off
+    @dates = create_dates
+  end
+
+  def create_dates
+    day_before_yesterday = (Time.zone.yesterday - 1).to_formatted_s(:iso8601)
+    yesterday = Time.zone.yesterday.to_formatted_s(:iso8601)
+    today = Time.zone.now.to_formatted_s(:iso8601)
+    tomorrow = Time.zone.tomorrow.to_formatted_s(:iso8601)
+    day_afer_tomorrow = (Time.zone.tomorrow + 1).to_formatted_s(:iso8601)
+    [day_before_yesterday, yesterday, today, tomorrow, day_afer_tomorrow]
+  end
+
+  def find_birthday_employees
+    birthdays = []
+    @employees.each do |employee|
```

Comment:
This `.each` / `.if` / `.append` looks like a `select`: https://ruby-doc.org/core-2.5.7/Enumerable.html#method-i-select

`[1,2,3,4,5].select { |num|  num.even?  }   #=> [2, 4]`